### ISSUE TITLE
Pi Zero W constantly crashes

### ISSUE THREAD

**Issue by garv3** [mKeRix/room-assistant#914]
I have installed Romm Assistant on a Pi Zero W as described here: https://www.room-assistant.io/guide/quickstart-pi-zero-w.html

It works just fine for a while but after two days or so the Pi becomes unavailable and cannot be accessed via SSH anymore. The only way to  fix this is to unplug and reconnect the power supply.

This makes the device highly unreliable and so unusable.

**Comment by Scott8586** [comment]
Unfortunately this is not unusual - it stems from the bluetooth and wifi having to share resources.  I find that having a wired interface to the pi zero helps a lot, I also reboot my pi zeros nightly.  Even with that I still have one zero that gets jammed up occasionally and the nightly reboot does not recover it.  Very frustrating.

**Comment by garv3** [comment]
Well, there has to be a solution for this problem.
Is this an OS level issue or is it caused by RA?
I am unable to use a wired Pi Zero at the location I need the Pi to be. So how can this be fixed? We can't just accept this behaviour.

**Comment by MagicMountainMan** [comment]
I went all in and got 8 RPiZeroW boards. I have the same issue where a few of them crash all the time and they have to be physically disconnected from power to be brought back. Frustrating for sure and hopefully there might be a time when this project is more stable, but it might just be a limitation of the $10 dev board that we're expecting too much out of.

**Comment by PeteBa** [comment]
You could try the just released v2.19.0 as that has some BLE stability improvements and helps some with RPiZWs. You need to add `instanceBeaconEnabled: false` in the bluetothLowEnergy section of your config file as this reduces the contention for the wifi/ble adapter.

**Comment by garv3** [comment]
`instanceBeaconEnabled: false` was already set.
I updated to the newest version, but I already decided to go another route:
I will only turn the Pi on when I know I might need the presence detection soon and I turn it off when I know I won't need it for some while. This process is automated and turning on is done by a simply activating a Smart Plug+. To turn it off a shutdown command is sent to the Pi via SSH and the Smart Plug+ is switched off 10 seconds later which should give the pi enough time to shut down. I hope, the Pi wont crash within those short periods of operation (usually no longer than 20 minutes).
